google/cloud/pubsub/internal/tracing_batch_callback.cc

```diff
@@ -123,6 +123,13 @@ class TracingBatchCallback : public BatchCallback {
   }
 
   void user_callback(MessageCallback::MessageAndHandler m) override {
+    std::unique_lock<std::mutex> lk(mu_);
+    auto spans = spans_by_ack_id_.find(m.ack_id);
```

Comment:
nit: `spans` is an odd name. It is certainly not plural. I would call it `it` (short for iterator).

Reply:
> It is certainly not plural.

Oops. My mistake. I should have read the code.